io/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Scala vs. Java is a good point; you can use Java APIs in your Scala code.

But, that is true for the entire SDK. Scala SDK may wrap a portion of the Java SDK, and the user can still use the other part. And, then, no other language can use it without a lot of trouble.

My general thinking is to make IOs SDK-specific, at least for now.